Views/Staff/Index.cshtml

```diff
@@ -9,29 +9,20 @@
 <p>
     <a asp-action="Create">Create New</a>
 </p>
```

Comment:
Search can be helpful, are you anticipating doing it in a different section of the app?